s/features/chat/components/xiome-chat/styles/chat-history.css.ts

```diff
@@ -2,10 +2,56 @@
 import {css} from "../../../../../framework/component.js"
 export default css`
 
-.history {}
+.history {
+	overflow-y: scroll;
```

Comment:
- maybe we can change `overflow-y` to `auto` or something like that, to only show scrollbars when necessary?